docs/examples.rst

```diff
@@ -0,0 +1,9 @@
+.. _examples:
+
+Examples of using skops
+=======================
+
+- Tabular Regression:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_tabular_classification.py>`_ is an example of using skops to serialize a tabular regression model and create a model card and a Hugging Face Hub repository.
+- Text Classification:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_text_classification.py>`_ is an example of using skops to serialize a text classification model and create a model card and a Hugging Face Hub repository.
```

Comment:
I would expect all auto-examples to be listed here:

https://skops--254.org.readthedocs.build/en/254/auto_examples/

At the same time, the paragraph on the start page of the skops page becomes a bit redundant with this new page:

> The following examples are good starting points: ...

([link](https://skops.readthedocs.io/en/stable/index.html))

Should we remove it in favor of just linking to this page? @adrinjalali @merveenoyan WDYT?

Reply:
I think that paragraph can stay since we can only link to a few examples where people can start, while also linking to this page with all examples listed.

Reply:
@adrinjalali I'm a little confused, are we wanting to link from the start page to this examples page?

Reply:
Yes we do @lazarust